pkg/option/config.go

```diff
@@ -1472,6 +1476,10 @@ type DaemonConfig struct {
 	// MTU is the maximum transmission unit of the underlying network
 	MTU int
 
+	// EnableRouteMTUForCNIChaining enables route MTU for pod netns when
+	// CNI chaining is used.
+	EnableRouteMTUForCNIChaining bool
+
```

Comment:
Could we add this field to d.mtuConfig instead? And add the flag to the `mtu` cell? We're trying to move away from adding flags to `DaemonConfig` and the MTU manager has already been hive-ified.